SU2_CFD/src/solvers/CHeatSolver.cpp

```diff
@@ -1002,7 +962,7 @@ void CHeatSolver::BC_ConjugateHeat_Interface(CGeometry *geometry, CSolver **solv
 
       if (geometry->nodes->GetDomain(iPoint)) {
 
-        Normal = geometry->vertex[val_marker][iVertex]->GetNormal();
+        su2double const* Normal = geometry->vertex[val_marker][iVertex]->GetNormal();
```

Comment:
This is probably more correct but in most places we have `const su2double*` or `const auto`.
Just because `su2double const*` and `su2double* const` are very different and may get confused...

Reply:
in f32f9ee